.github/workflows/ci.yml

```diff
@@ -126,6 +126,10 @@ jobs:
           # to save a skipped resource list
           SKIPPED_RESOURCES_CSV: ../_output/skipped_resources.csv
 
+      - name: Show diff
+        if: failure() && steps.check-diff.outcome == 'failure'
+        run: git diff
```

Comment:
Hmm, wouldn't this help already? https://github.com/upbound/build/pull/168

Reply:
Thank you for the pointer @turkenh. I checked what's implemented there. Looks like we would still be missing the file content diff output I want to achieve with that change.

A couple of times, I've run into situations where I did not know what the content diff was as a result of `make generate` run. We have also discussed offline whether it would make sense to make this functionality part of the build submodule (i.e., if a `git diff` detects changes in the files, instead of reporting a status output, we could report a content output). The concern was to keep the content diff report separate in a different job step then the `Check Diff` step.